tests/files/mpeg.rs

```diff
@@ -149,7 +149,7 @@ fn save_to_id3v2() {
 	tag.set_title("title".to_string());
 
 	file.rewind().unwrap();
-	tag.save_to(&mut file).unwrap();
+	tag.save_to(&mut file, WriteOptions::new()).unwrap();
```

Comment:
Use `WriteOptions::default()` to make this more explicit? I even prefer the generic `Default::default()` if the type could be inferred.

Reply:
Yeah, `default()` is probably better.